sw/device/lib/testing/usb_testutils_controlep.c

```diff
@@ -10,7 +10,7 @@
 #include "sw/device/lib/testing/usb_testutils.h"
 
 // Device descriptor
-static uint8_t dev_dscr[] = {
+static const uint8_t dev_dscr[] = {
```

Comment:
Const naming convention.
```suggestion
static const uint8_t kDevDscr[] = {
```